Solutions/NonameSecurity/Data Connectors/Connector_RESTAPI_NonameSecurity.json

```diff
@@ -0,0 +1,104 @@
+{
+    "id": "NonameSecurityMicrosoftSentinel",
+    "title": "Noname Security for Microsoft Sentinel",
+    "publisher": "Noname Security",
+    "descriptionMarkdown": "Noname Security solution to POST data into a Microsoft Sentinel SIEM workspace via the Azure Monitor REST API",
+    "graphQueries": [
+        {
+            "metricName": "Noname API Security Alert",
+            "legend": "NonameAPISecurityAlert_CL",
+            "baseQuery": "NonameAPISecurityAlert_CL"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Noname API Security Alerts",
+            "query": "NonameAPISecurityAlert_CL\n| where TimeGenerated >= ago(1d)\n"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "NonameAPISecurityAlert_CL",
+            "lastDataReceivedQuery": "NonameAPISecurityAlert_CL\n | summarize Time = max(TimeGenerated)\n | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "NonameAPISecurityAlert_CL\n| summarize LastAlertReceived = max(TimeGenerated)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1,
+        "isPreview": true
+    },
+    "permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "write": true,
+                    "read": true,
+                    "delete": true
+                }
+            },
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces/sharedKeys",
+                "permissionsDisplayText": "read permissions to shared keys for the workspace are required.",
```

Comment:
Hey @nniryanb, please replace this line 52 with the provided text, this will pass the validation.

"permissionsDisplayText": "read permissions to shared keys for the workspace are required. [See the documentation to learn more about workspace keys](https://docs.microsoft.com/azure/azure-monitor/platform/agent-windows#obtain-workspace-id-and-key).",